mlops/bicep/modules/computes/open_new_aks_with_confcomp.bicep

```diff
@@ -0,0 +1,149 @@
+// This BICEP script will provision an AKS cluster with confidential computes
+// attached to a given AzureML workspace, without any specific security settings.
+
+// IMPORTANT: This setup is intended only for demo purpose. The data is still accessible
+// by the users when opening the storage accounts, and data exfiltration is easy.
+
+// NOTE: this can take up to 15 minutes to complete
+
+// resource group must be specified as scope in az cli or module call
+targetScope = 'resourceGroup'
+
+// required parameters
+@description('Name of AzureML workspace to attach compute+storage to.')
+param machineLearningName string
+
+@description('The region of the machine learning workspace')
+param machineLearningRegion string = resourceGroup().location
+
+@description('The name of the Managed Cluster resource.')
+param aksClusterName string
+
+@description('How to name this compute in Azure ML')
+param amlComputeName string = aksClusterName
+
+@description('Specifies the location of the compute resources.')
+param computeRegion string
+
+@description('Optional DNS prefix to use with hosted Kubernetes API server FQDN.')
+@maxLength(54)
```

Comment:
great find, I didn't know about this maxlength functionality, or about the min/maxValue